### ISSUE TITLE
in 19.3.1 flatbuffers is not automatically installed using pip install

### ISSUE THREAD

**Issue by jberends** [crossbario/autobahn-python#1141]
When I just installed autobahn (yeah I know, released 25 minutes ago) I got the following import error:

```
[...]
    from autobahn.wamp import protocol, auth
  File "]...]venv/lib/python3.6/site-packages/autobahn/wamp/protocol.py", line 37, in <module>
    from autobahn.wamp import message
  File "[...]/venv/lib/python3.6/site-packages/autobahn/wamp/message.py", line 37, in <module>
    from autobahn.wamp import message_fbs
  File "[...]/venv/lib/python3.6/site-packages/autobahn/wamp/message_fbs.py", line 29, in <module>
    import flatbuffers
ModuleNotFoundError: No module named 'flatbuffers'
```
It seems that pip install autobahn does not install the flatbuffers module. Checked `setup.py` quickly but could not find it there (fast glance)

**Comment by oberstet** [comment]
uups:( should be addressed in https://github.com/crossbario/autobahn-python/pull/1142 which we'll release right away ..

> note: the CI missed that, as we run with "full deps installed" there

**Comment by danieltatrai** [comment]
verified!

**Comment by oberstet** [comment]
fixed in v19.3.2: https://pypi.org/project/autobahn/19.3.2/ via https://github.com/crossbario/autobahn-python/pull/1142

```
(cpy372_3) oberstet@intel-nuci7:~$ pip install autobahn==19.3.2
Collecting autobahn==19.3.2
  Downloading https://files.pythonhosted.org/packages/ea/65/e474985b604f91e15b6d07a83ab99650f3e43791c9711f72ad32a29b24e2/autobahn-19.3.2-py2.py3-none-any.whl (387kB)
    100% |████████████████████████████████| 389kB 3.3MB/s 
Requirement already satisfied: txaio>=18.8.1 in ./cpy372_3/lib/python3.7/site-packages (from autobahn==19.3.2) (18.8.1)
Requirement already satisfied: six>=1.11.0 in ./cpy372_3/lib/python3.7/site-packages (from autobahn==19.3.2) (1.12.0)
Installing collected packages: autobahn
Successfully installed autobahn-19.3.2
You are using pip version 18.1, however version 19.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
(cpy372_3) oberstet@intel-nuci7:~$ python
Python 3.7.2 (default, Feb 25 2019, 08:26:41) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import txaio
>>> txaio.use_asyncio()
>>> from autobahn.wamp import protocol, auth
>>> 
``

**Comment by jberends** [comment]
Thanks; great responsiveness!
